core/server/master/src/main/java/alluxio/master/file/meta/LockedInodePath.java

```diff
@@ -221,6 +221,19 @@ public List<InodeView> getInodeViewList() {
     return mLockList.getLockedInodeViews();
   }
 
+  /**
+   * @return a copy of the list of existing inodes and empty inodes
+   */
+  public List<InodeView> getInodeViewListWithEmptyInodes() {
+    List<InodeView> inodes = getInodeViewList();
+    int i = inodes.size();
+    while (i < mPathComponents.length) {
```

Comment:
Is it possible to have more than 1 empty inodes? If not, explain in the comment why.